pkgs/development/perl-modules/generic/default.nix

```diff
@@ -37,6 +37,6 @@ stdenv.mkDerivation (
     name = "perl${perl.version}-${name}";
     builder = ./builder.sh;
     nativeBuildInputs = nativeBuildInputs ++ [ (perl.dev or perl) ];
-    inherit perl;
+    perl = buildPackages.perl;
```

Comment:
is it used somehow?
Both in this fix and in general (I suspect this ```perl```-inside-packages can be safely removed)

Reply:
it's used at the end of this line https://github.com/NixOS/nixpkgs/blob/33ef2829b663945d0d7ae702a6f5f5f342760401/pkgs/development/perl-modules/generic/builder.sh#L25 to give perl's Makefile a version of perl that can be used on the build platform (during build time).